components/layout_2020/fragments.rs

```diff
@@ -193,7 +228,10 @@ impl AnonymousFragment {
         AnonymousFragment {
             debug_id: DebugId::new(),
             rect,
-            children,
+            children: children
+                .into_iter()
+                .map(|fragment| ArcRefCell::new(fragment))
+                .collect(),
```

Comment:
Please add a commit that changes this method to take `Vec<ArcRefCell<Fragment>>` and push `ArcRefCell::new` as early as necessary in various callers to avoid the intermediate `Vec`

And similarly in:
* `BoxFragment::new`
* `flow/root.rs` for `FragmentTreeRoot::children`
* `positioned.rs` for `new_fragment.children.extend(…)`

Reply:
Thanks for the review.

I ended up using intermediate `Vec`s here in order to avoid a lot of mutable borrows during the creation of each `Fragment`. I wasn't sure if it made sense performance-wise to do some number of mutable borrows per-fragment versus creating a temporary vector for `BoxFragment`s and `AnonymousFragment`s with children. What's your opinion on this?

Reply:
Okay. After our discussion on Matrix, I stuck with just the original commit. Thanks for the reviews.